examples/tutorials/04_cubes_and_objects/03_go_to_object_test.py

```diff
@@ -22,13 +22,12 @@
 '''
 
 import asyncio
-import sys
 
 import cozmo
 from cozmo.util import degrees, distance_mm
 
 
-def go_to_object_test(robot):
+def go_to_object_test(robot: cozmo.robot.Robot):
```

Comment:
Looks like this script used to support having a KeyboardInterrupt. Does it still? If yes, we should have this line:

print("Press CTRL-C to quit")